tools/install.sh

```diff
@@ -254,6 +255,7 @@ main() {
       --unattended) RUNZSH=no; CHSH=no ;;
       --skip-chsh) CHSH=no ;;
       --keep-zshrc) KEEP_ZSHRC=yes ;;
+      --skip-git-clone) SKIP_GIT_CLONE=yes ;;
```

Comment:
What does this do?

Reply:
If for some reason user clone the repo manually before running the script, he can skip git clone process. For example, user wants to share a same clone among multiple linux users.